deploy/os/che-workspace-controller-cert-gen-deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: che-workspace-controller-cert-gen
+  namespace: che-workspace-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: che-workspace-controller-cert-gen
+  template:
+    metadata:
+      labels:
+        app: che-workspace-controller-cert-gen
+      annotations:
+        kubectl.kubernetes.io/restartedAt: ""
+    spec:
+      serviceAccountName: che-workspace-controller
```

Comment:
does it need SA or it's just for a namespace to be mount?
If so - isn't it better just to provision namespace value with env var and valueFrom: metadata.namespace?

Reply:
It needs the service account for the configmap/service to be created. Otherwise it just ends up erroring. Now that I've fixed the logging issue it will show the error in console if you don't have the service account